script/brave_license_helper.py

```diff
@@ -28,7 +28,8 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
     prune_paths.update([
         # Formerly external Brave code which has moved to brave-core
         # (i.e these are already covered by the Brave Browser license notice).
-        os.path.join('brave', 'third_party', 'challenge_bypass_ristretto_ffi'),
+        os.path.join('brave', 'components', 'challenge_bypass_ristretto',
+                     'rust', 'cxx'),
```

Comment:
I believe you can omit this altogether, now that the cxx package has moved under `components`. cc @fmarier

Reply:
agreed, removed these lines